include/mbgl/map/map.hpp

```diff
@@ -141,6 +142,9 @@ class Map : private util::noncopyable {
     LatLng latLngForProjectedMeters(const ProjectedMeters&) const;
     ScreenCoordinate pixelForLatLng(const LatLng&) const;
     LatLng latLngForPixel(const ScreenCoordinate&) const;
+    
+    // Transform
+    const TransformState& getTransform() const;
```

Comment:
Hopefully we’ll be able to expose just the projection matrix rather than the entire `TransformState` object.